src/interfaces/slack_bot/src/utils/formatRagCitations.test.ts

```diff
@@ -0,0 +1,156 @@
+import { describe, expect, test } from 'vitest';
```

Comment:
Please change all this test data

Reply:
I have changed all of the test data in this file.